server/webapp/WEB-INF/rails.new/webpack/views/dashboard/models/pipeline_list_vm.js

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
can we define the view model under `views/dashboard/models`?
Similar to [trigger with options VM](https://github.com/gocd/gocd/blob/master/server/webapp/WEB-INF/rails.new/webpack/views/dashboard/models/trigger_with_options_vm.js)?

Reply:
Isn’t that where this file is already?

`views/dashboard/models/pipeline_list_vm.js`

Reply:
Whoops! Yes you are right :)